### PR TITLE
Update common3.json

### PR DESCRIPTION
**Pull request by FalKoopa**
Added more translations, till id 61418

I couldn't translate some id's or I'm unsure about them, those are below, with my notes:

60238 Characteristic Random Number
60244 Capture Random Number
60329 M4 Startup Disk after obtaining
60378
60379
60908 Trainer name
60911 Trainer name
61106 - 61110 Trainer name
61132 - 61139 Trainer names
61150 - 61158 Trainer names
61160 - 61162 Trainer names
61165 - 61171 Trainer names
61173 - 61174 Trainer names
61233 Some Status condition
61280 Neo Hull?
61335 Hizuki Grandfather
61347 - 61348 Some Trainer Class
61361 Some Trainer Class
61375 Trainer Name
61377 Trainer Name
61381 Sheriff Assistant

### CODE REVIEW

**Comment by StarsMmd** [comment]
60238 PID RNG Roll
60244 Capture RNG Roll
60329 M4: After obtaining the D-Disk
(M4 is the code for the under)
60378 PR footage 1: 1v1 double battle
60379 PR footage 2: 1v1 double battle
60911 Weeg
61106 - 61110 Ein
61132 - 61139 Jomas, Delan, Nella, Ston
61150 - 61158 These trainers were cut and have untranslated names
(61152-61157 are the shadow aipom brothers https://tcrf.net/Pok%C3%A9mon_Colosseum)
61160 - 61162 Also cut and untranslated
61165 - 61171 More cut trainers, though 61170 was translated (Loget)
61173 - 61174 Cut but translated: Malpes, Longo
(Malpes is "Malpes the riddle lover" https://tcrf.net/Pok%C3%A9mon_Colosseum)
61233 "Seeded" (as in leech seed)
61280 This is a weird one. There are animation files with this name and they're named the way the parts of a status move would be named. However I can't find any move with that name. Maybe a cut move? Needs investigation.
61335 Eagun
(Hizuki is Rui's development name)
61347 - 61348 Female/Male body builder? 
61361 Chaser
61375 Cut and untranslated
61377 This trainer doesn't even appear in the game files at all. Likely cut very early. Fitting that their name literally means "mythical"
61381 Officer Johnson

File: common3.json [hunk 1]
```diff
@@ -2947,1423 +2947,1423 @@
...
     },
     {
       "id" : 61039,
-      "text" : "シャイナ3"
+      "text" : "Liaks 3"
     },
     {
-      "id" : 61040,
-      "text" : "ウィス1"
+      "id" : ,
```
Comment (aldelaro5) [inline common3.json]: removed id, this must be reverted

**Review by aldelaro5 — CHANGES_REQUESTED** [review]
There are added spaces between " and : of the id, but after checking the file, it seems that most of them were like this so whatever autoformatter you used actually improved the consistency so it's fine.

As for the strings @StarsMmd suggested, you should open another pr with them (or more if you feel like it).

The removed id however must be reverted before I merged, otherwise LGTM

File: common3.json (hunk 1 above)
Comment (FalKoopa) [reply common3.json]: The added spaces between " and : were actually there originally. I was using Visual Studio which automatically removed them so I re-added them in this pull-request.

I just pushed a new commit.
I've added the translations for those as given by Starmmd. Thank you.
And I also added that removed id. No idea how that happened.

**Review by aldelaro5 — APPROVED** [review]

**Pull request merged**